srml/eth-backing/src/lib.rs

```diff
@@ -205,9 +205,9 @@ decl_module! {
 					.clone()
 					.to_bytes()
 					.ok_or("Convert to Bytes - FAILED")?;
-				let decoded_sub_key = hex::decode(&raw_sub_key).map_err(|_| "Decode Address - FAILED")?;
+//				let decoded_sub_key = hex::decode(&raw_sub_key).map_err(|_| "Decode Address - FAILED")?;
 
-				T::DetermineAccountId::account_id_for(&decoded_sub_key)?
+				T::DetermineAccountId::account_id_for(&raw_sub_key[..])?
```

Comment:
if the function take `&[u8]`, a reference on `Vec<u8>` will convert to `slice` automatically.